book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] We can modify this method to avoid duplicating the `@rentals`.

  ```
  def add_rentals(date, person)
    Rental.new(date, self, person)
  end
  ```